### PR TITLE
IFD-283/Fix migration errors and update html-to-markdown to 5.0.0

### PR DESCRIPTION
**Pull request by 186dk**

### CODE REVIEW

File: src/Commands/WaContent.php [hunk 1]
```diff
@@ -330,32 +330,37 @@ private function saveCompositeContents($postId, ?Collection $compositeContents)
 
         $content = $compositeContents
             ->map(function ($compositeContent) use ($postId) {
-                if ($compositeContent->type === 'text_item') {
+                if ($compositeContent->type === 'text_item' && ! empty($compositeContent->text ?? null)) {
+                var_dump($compositeContent);
```
Comment (madspaaskesen) [inline src/Commands/WaContent.php:334]: Small var_dump here... think it should be removed...

File: src/Commands/WaContent.php [hunk 2]
```diff
@@ -330,32 +330,37 @@ private function saveCompositeContents($postId, ?Collection $compositeContents)
...
                     return [
                         'body'           => HtmlToMarkdown::parseHtml($compositeContent->text),
                         'locked_content' => false,
                         'acf_fc_layout'  => $compositeContent->type,
                     ];
                 }
-                if ($compositeContent->type === 'image') {
+                if ($compositeContent->type === 'image' && ! empty($compositeContent->url ?? null)) {
+                    $leadImage = $compositeContent->lead_image ?? false;
+                    ( ! $leadImage && $this->site->product_code === "IFO") ? $displayHint = 'sm' : $displayHint = "default";
```
Comment (madspaaskesen) [inline src/Commands/WaContent.php:343]: think
$displayHint = ( ! $leadImage && $this->site->product_code === "IFO") ? 'sm' : 'default';
is smaller

File: src/Commands/WaContent.php (hunk 1 above)
Comment (186dk) [reply src/Commands/WaContent.php:334]: yes

**Review by madspaaskesen — APPROVED** [review]

File: src/Commands/WaContent.php (hunk 2 above)
Comment (186dk) [reply src/Commands/WaContent.php:342]: ifo will have small image (sm), if it is not lead image

Comment (186dk) [reply src/Commands/WaContent.php:342]: Yes, that is better

**Review by madspaaskesen — APPROVED** [review]

**Pull request merged**